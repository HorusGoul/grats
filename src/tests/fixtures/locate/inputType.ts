// Locate: User
/** @gqlInput */
type User = {
  /** @gqlField */
  name: string;
};
