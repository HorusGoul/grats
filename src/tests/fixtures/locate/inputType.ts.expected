-----------------
INPUT
----------------- 
// Locate: User
/** @gqlInput */
type User = {
  /** @gqlField */
  name: string;
};

-----------------
OUTPUT
-----------------
src/tests/fixtures/locate/inputType.ts:3:6 - error: Located here

3 type User = {
       ~~~~
